mlrun/builder.py

```diff
@@ -39,11 +39,13 @@ def make_dockerfile(
     commands=None,
     source=None,
     requirements=None,
-    workdir="/mlrun",
+    workdir=None,
     extra="",
     user_unix_id=None,
     enriched_group_id=None,
 ):
+    tmpdir = tempfile.mkdtemp()
```

Comment:
only call that if workdir was not provided. remember that it is up to the callee to remove the tempdir, so running that many times will leave many stale temp dirs.